src/components/VirtualGrid.vue

```diff
@@ -9,14 +9,23 @@
 			class="grid-filler grid-filler--top"
 			role="none" />
 
-		<!-- files list -->
-		<component :is="component(item)"
-			v-for="(item, index) in shownList"
-			:key="item.fileid"
-			:ref="`item-${index}`"
-			:class="`row-${getRowNumber(index)}`"
-			:list="list"
-			v-bind="props(item)" />
+		<template v-for="(item, index) in shownList">
+			<div v-if="index == 0 || getFormatedDate(item.lastmod) != getFormatedDate(shownList[index - 1].lastmod)"
```

Comment:
Ideally this should be in its own method so we can have a clear view on the conditions we split things :)

Reply:
Let's see after making good logic.